CHANGELOG.md

```diff
@@ -20,19 +20,31 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 
 ### Features
 
-- Allow user to query foreign applications accounts using the `appID` field.
-- Remove limits from `Runtime` for amount of apps/assets one account can create/opt-in to.
-- Add `getGenesisHashFromName(name: string)` utility function to `@algo-builder/web`.
-- Add `mainnetGenesisHash`, `testnetGenesisHash`, `betanetGenesisHash`, `runtimeGenesisHash` constants.
+Algob:
+
 - Add `parseABIContractFile(pathToFilePath)` method to `Runtime` and `Deployer`. If the currently used network is defined in the ABI file additonal field `appID` will be added to a contract.
-- Add support for TEALv8 opcode execution in runtime. The `MaxTEALVersion` is now `8`.
-- Added `allowMultipleAccounts` parameter to `connectToMyAlgo` method of `WebMode` to give user the flexibility to allow multi accounts login using MyAlgo Wallet.
 - Added arguments feature in `yarn algob run` cli. The new format is `yarn alob run --script script1.js arg1 arg2`.
+
+Runtime:
+
 - Add `TxFieldEnum` for transaction fields in `runtime`.
 - Add enums for opcode fields in `runtime`.
-- Add reusable workflow in GitHub action.
 - Add one batch for each package in GitHub action which uses reusable workflow.
 - Add guide about blocks in `Runtime`.
+- Remove limits from `Runtime` for amount of apps/assets one account can create/opt-in to.
+- Add support for TEALv8 opcode execution in runtime. The `MaxTEALVersion` is now `8`.
+- Allow user to query foreign applications accounts using the `appID` field.
+
+Web:
+
+- Add `mainnetGenesisHash`, `testnetGenesisHash`, `betanetGenesisHash`, `runtimeGenesisHash` constants.
+- Added `allowMultipleAccounts` parameter to `connectToMyAlgo` method of `WebMode` to give user the flexibility to allow multi accounts login using MyAlgo Wallet.
+- Add `web.mkMultisigEncodedTx` helper function to create multisig encoded transaction.
```

Comment:
wouldn't it be `tx.mkMultisigEncodedTx` since we export the txn file as `export * as tx from "./lib/txn";`

Reply:
right, updating it now.

Reply:
```suggestion
- Add `tx.mkMultisigEncodedTx` helper function to create multisig encoded transaction.
```